recipes-debian/mhash/mhash_debian.bb

```diff
@@ -0,0 +1,34 @@
+inherit debian-package
+require recipes-debian/sources/mhash.inc
+
+LICENSE = "LGPLv2"
```

Comment:
Some files are licensed under LGPLv2.1, so I think "LGPLv2+" is appropriate.

Reply:
Thank you for the detailed check. Updated.